### PR TITLE
feat(components): update typography to v9

### PR DESCRIPTION
**Pull request by geotrev**
## Description

Updates `react-typography` to `v9`.

## Detail

Update pages using typography components with colors:
- [x] Demonstrate variable keys in `Span` color examples
- [x] Updates examples (`Timeline`, `Draggable`) using `Span` with variable keys
- [x] Updates `Grid` subcomponent properties

## Checklist

- ~~ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)~~
- ~~:black_nib: copy updates are approved (add the content strategist as a reviewer)~~
- [x] :link: considered opportunities for adding cross-reference URLs (grep for keywords)
- ~~:wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver~~
- ~~:memo: tested in Chrome, Firefox, Safari, Edge~~

### CODE REVIEW

File: src/examples/typography/code/Color.tsx [hunk 1]
```diff
@@ -8,27 +8,27 @@
...
+    <Grid.Col sm={4} textAlign="center">
       <Code hue="green">Veggies es bonus</Code>
-    </Col>
+    </Grid.Col>
     <StyledCol sm={4} textAlign="center">
       <Code hue="yellow">Veggies es bonus</Code>
     </StyledCol>
     <StyledCol sm={4} textAlign="center">
       <Code hue="red">Veggies es bonus</Code>
     </StyledCol>
```
Comment (ze-flo) [inline src/examples/typography/code/Color.tsx:30]: Making a mental note to update docs when `props.hue` supports variables.

ex: `<Span hue="foreground.subtle">`

**Review by ze-flo — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     "@zendeskgarden/react-tags": "8.76.3",
     "@zendeskgarden/react-theming": "9.0.0-next.19",
     "@zendeskgarden/react-tooltips": "^9.0.0-next.20",
-    "@zendeskgarden/react-typography": "9.0.0-next.19",
+    "@zendeskgarden/react-typography": "^9.0.0-next.20",
```
Comment (jzempel) [inline package.json:58]: Please remove range dependencies

File: src/examples/typography/code/Color.tsx (hunk 1 above)
Comment (jzempel) [reply src/examples/typography/code/Color.tsx:30]: > Making a mental note to update docs when `props.hue` supports variables.
> 
> ex: `<Span hue="foreground.subtle">`

In this case, light/dark mode will provide the desired treatment for the hues provided. No change needed.

**Review by jzempel — CHANGES_REQUESTED** [review]
Please note there will be more work needed on typography once the v9 sequence is further along due to the fact that `Typescale` will be the new, preferred import for typography sizes.

**Comment by geotrev** [comment]
> Please note there will be more work needed on typography once the v9 sequence is further along due to the fact that Typescale will be the new, preferred import for typography sizes.

Ah, yes, I see that in the sequence. Thanks for the heads up. I'll make a note for that!

File: src/examples/typography/span/Hue.tsx [hunk 3]
```diff
@@ -19,9 +19,9 @@ const Example = () => (
       <MD tag="span">
         Water spinach arugula pea tatsoi aubergine spring onion bush tomato kale radicchio turnip
         chicory salsify pea sprouts fava bean.{' '}
-        <Span hue="fuschia">Dandelion zucchini burdock yarrow chickpea dandelion</Span> sorrel
-        courgette turnip greens tigernut soybean radish artichoke wattle seed endive groundnut
-        broccoli arugula.
+        <Span hue="foreground.primary">Dandelion zucchini burdock yarrow chickpea dandelion</Span>{' '}
+        sorrel courgette turnip greens tigernut soybean radish artichoke wattle seed endive
+        groundnut broccoli arugula.
```
Comment (ze-flo) [inline src/examples/typography/span/Hue.tsx:24]: Should we show all 3 possibilities for setting the text color: variables, hue, and HEX?

Comment (geotrev) [reply src/examples/typography/span/Hue.tsx:24]: I'm open to it. I figured we didn't want to just because we encourage avoiding hard coded hexes.

Comment (jzempel) [reply src/examples/typography/span/Hue.tsx:24]: For the purposes of this PR, let's leave the essential content as-is. We can look at example enhancements later. But we probably don't want to encourage random hex usage, in general.

**Review by jzempel — APPROVED** [review]
Nice :tada:

**Pull request merged**